.github/workflows/continous-integration.yml

```diff
@@ -552,6 +552,9 @@ jobs:
         uses: actions/checkout@v3
         if: github.event_name != 'pull_request'
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```

Comment:
Any reason you're opting for v1? Looks like we're up to [v2.1.0](https://github.com/docker/setup-qemu-action/releases) now - and it'd be [preferable](https://www.notion.so/rasa/Third-Party-GitHub-Actions-Security-Advice-d6815bab2f664b7996c22960214021a6) if you could pin to the hash instead of a version eg:

```
uses: docker/setup-qemu-action@e81a89b1732b9c48d79cd809d8d81d79c4647a18
```